src/main/java/org/liliya/hotelapp/controller/AppServlet.java

```diff
@@ -0,0 +1,203 @@
+package org.liliya.hotelapp.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.liliya.hotelapp.configuration.Configuration;
+import org.liliya.hotelapp.model.Apartment;
+import org.liliya.hotelapp.model.Client;
+import org.liliya.hotelapp.persistence.StatePersistence;
+import org.liliya.hotelapp.persistence.StatePersistenceImpl;
+import org.liliya.hotelapp.service.ApartmentService;
+import org.liliya.hotelapp.service.ApartmentServiceImpl;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+
+public class AppServlet extends HttpServlet {
```

Comment:
In general good job)) but, can we sptil the logic? here we have a lof of responsibility 
maybe we can create different Servlets in general?

Reply:
created different servlets